.github/workflows/publish-ci.yml

```diff
@@ -113,5 +113,5 @@ jobs:
         if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.DEEPHAVENBOT_NPM_TOKEN }}
-        run: npm publish --tag latest web/client-api/types/build/deephaven-jsapi-types-*.tgz
+        run: npm publish --tag v0.33 web/client-api/types/build/deephaven-jsapi-types-*.tgz
```

Comment:
To make sure I understand a bit more: a _tag_ here is _not_ the same as npm "version" - it's an additional qualifier that points to a version?

Can a single version have multiple tags?

Looking at https://www.npmjs.com/package/@deephaven/jsapi-types?activeTab=versions, I see the tags `latest`, `nightly`, and `canary`. Are we planning to introduce a new convention for all npm releases, with a tag that is `v<major>.<minor>`?

https://docs.npmjs.com/adding-dist-tags-to-packages seems to recommend _against_ this:
> We recommend avoiding dist-tags that start with a number or the letter "v".

Reply:
Correct; we do the v0.x tagging on the web-client-ui packages: https://www.npmjs.com/package/@deephaven/code-studio?activeTab=versions
But it could be any arbitrary name. Yea npm recommends against it, but not sure what else to name them? We could give them the same name as the Enterprise releases they coincide with (e.g. `vplus`), and maybe that would be a better fit (though ironically still wouldn't be recommended, since it starts with `v`)?